cvat/settings/base.py

```diff
@@ -562,6 +562,8 @@ def add_ssh_keys():
         'JobStage': 'cvat.apps.engine.models.StageChoice',
         'StorageType': 'cvat.apps.engine.models.StorageChoice',
         'SortingMethod': 'cvat.apps.engine.models.SortingMethod',
+        'WebhookType': 'cvat.apps.webhooks.models.WebhookTypeChoice',
+        'WebhookContentType': 'cvat.apps.webhooks.models.WebhookContentTypeChoice',
```

Comment:
Can't we rename the classes themselves? Why do it here?

Reply:
This is done here, because this component is responsible for such renames for better look of the generated schema. The class on the server can be renamed independently, if it is needed.